front/src/components/corpus/CorpusCreate.jsx

```diff
@@ -52,12 +52,12 @@ export default function CorpusCreate({ onSubmit }) {
         })
         onSubmit()
         setToast({
-          text: `New corpus created`,
+          text: {t('corpus.create.toastSuccess')},
```

Comment:
@gromettoclara ici, comme on n'est pas dans une balise React, y'a pas besoin de mettre les `{` / `}`. Ça produit une erreur de syntaxe JS.

```suggestion
          text: t('corpus.create.toastSuccess'),
```